mne/io/utils.py

```diff
@@ -312,3 +312,24 @@ def _construct_bids_filename(base, ext, part_idx):
     if dirname:
         use_fname = op.join(dirname, use_fname)
     return use_fname
+
+
+def _get_cart_ch_coords_from_inst(chs, drop_chs=None):
```

Comment:
I don't think this name reflects what it does -- our channels are already in Cartesian coordinates, we just use a RAS definition for our dimensions and it looks like EEGLAB uses ALS (based on the changes). So I would name this `_get_als_coords_from_ras` or `_get_eeglab_coords_from_inst` or something

Reply:
I decided on `_get_als_coords_from_chs`.